test/test_optim.py

```diff
@@ -432,7 +432,7 @@ def fwd_bwd(optim, w, b, i):
             optimizer_c.load_state_dict(deepcopy(optimizer.state_dict()))
 
             # Run both optimizers in parallel
-            for i in range(10):
+            for _ in range(10):
```

Comment:
We can enable lint for these new files if you want

Reply:
We can do this in another PR; I'm not too concerned for this one. Lint would likely help in the future though.